cmake/CMakeLists.txt

```diff
@@ -1938,6 +1971,11 @@ if (CMAKE_SYSTEM_NAME STREQUAL "Android")
   list(APPEND onnxruntime_EXTERNAL_LIBRARIES log)
 endif()
 
+if (onnxruntime_USE_XNNPACK)
```

Comment:
what if (onnxruntime_BUILD_WEBASSEMBLY AND NOT onnxruntime_ENABLE_WEBASSEMBLY_THREADS) ?

is pthreadpool still need?

Reply:
If you are asking the binary code, yes.